ingestion_program/datasets.py

```
@@ -28,6 +28,10 @@
     "https://www.codabench.org/datasets/download/b9e59d0a-4db3-4da4-b1f8-3f609d1835b2/"
 )
 
+ZENODO_API = "https://zenodo.org/api/deposit/depositions"
+ACCESS_TOKEN = os.getenv("ZENODO_ACCESS","")   
+DEPOSITION_ID = os.getenv("ZENODO_ID","")
```

What do you think if the access token and deposition id are arguments to the function where they are used? This way we can raise errors if these are not provided, otherwise these  will be used to download the dataset.

---

Ignore that for, I will change that block tomorrow, when dataset because published.